### PR TITLE
fix(templates): fix template auto creation

### PR DESCRIPTION
**Pull request by petrjasek**
which was failing after first run, because it was using id from
template for item. now this field is removed.

SD-2099

### CODE REVIEW

File: server/apps/templates/content_templates.py [hunk 1]
```diff
@@ -143,16 +143,15 @@ def set_template_timestamps(template, now):
...
 def create_item_from_template(template):
-    """Create item in production using data from given template.
+    """ Create item using data from template.
 
     :param dict template
     """
-    production = superdesk.get_resource_service('archive')
     item = {key: value for key, value in template.items() if key in metadata_schema}
+    item['template'] = item.pop('_id')
     item['state'] = SCHEDULED_ITEM_STATE
```
Comment (sivakuna-aap) [inline server/apps/templates/content_templates.py]: Can we use constants for `state`, `archive`? Also, `SCHEDULED_ITEM_STATE` is defined in `item.py.CONTENT_STATE`

Comment (petrjasek) [inline server/apps/templates/content_templates.py]: right updated it. it shouldn't be scheduled state after all, but probably submitted - it's a new item on a desk that needs some work, not scheduled for publishing.

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**